tempodb/encoding/v2/appender.go

```diff
@@ -49,9 +46,9 @@ func (a *appender) Append(id common.ID, b []byte) error {
 		return err
 	}
 
-	a.hash.Reset()
-	_, _ = a.hash.Write(id)
-	hash := a.hash.Sum64()
+	hasher := xxhash.New()
```

Comment:
fwiw - I think we could keep the static hash for writes and just alloc a new one for reads.

Reply:
yeah, that's fair. it's a small improvement, but this code is called constantly.

Reply:
I have updated the code